aiozipkin/transport.py

```diff
@@ -39,7 +39,9 @@ def send(self, record):
 
         # TODO: add status code check
         try:
-            async with self._session.post(self._address, json=data) as resp:
+            headers = {'Content-Type': 'application/json'}
+            async with self._session.post(self._address, json=data,
```

Comment:
@asvetlov is this expected, I thought client sets `Content-Type`to json if `json` parameter is used.